ontology/uco/tool/tool.ttl

```diff
@@ -326,17 +271,59 @@ tool:buildInformation
 	.
 
 tool:buildLabel
-	a owl:DatatypeProperty ;
+	a
+		owl:DatatypeProperty ,
+		owl:DeprecatedProperty
+		;
 	rdfs:label "buildLabel"@en ;
 	rdfs:comment "Relevant label for a particular build of a particular software."@en ;
 	rdfs:range xsd:string ;
+	rdfs:seeAlso
+		core:tag ,
+		observable:SoftwareBuildFacet
+		;
+	.
+
+tool:buildLabel-subjects-shape
+	a sh:NodeShape ;
+	rdfs:seeAlso
+		core:tag ,
+		observable:SoftwareBuildFacet
+		;
+	sh:property [
+		sh:maxCount "0"^^xsd:integer ;
+		sh:message "uco-tool:buildLabel is depreacted.  Please instead use the path uco-core:hasFacet (linking a uco-observable:SoftwareBuildFacet) / uco-core:tag."@en ;
+		sh:path tool:buildLabel ;
+	] ;
+	sh:targetSubjectsOf tool:buildLabel ;
 	.
 
 tool:buildOutputLog
-	a owl:DatatypeProperty ;
+	a
+		owl:DatatypeProperty ,
+		owl:DeprecatedProperty
+		;
 	rdfs:label "buildOutputLog"@en ;
 	rdfs:comment "The output log of the build process for a software."@en ;
 	rdfs:range xsd:string ;
+	rdfs:seeAlso
+		observable:SoftwareBuildFacet ,
+		observable:buildOutputLog
+		;
+	.
+
+tool:buildOutputLog-subjects-shape
+	a sh:NodeShape ;
+	rdfs:seeAlso
+		observable:SoftwareBuildFacet ,
+		observable:buildOutputLog
+		;
+	sh:property [
+		sh:maxCount "0"^^xsd:integer ;
+		sh:message "uco-tool:buildOutputLog is deprecated.  Please instead use the path uco-core:hasFacet (linking a uco-observable:SoftwareBuildFacet ) / uco-observable:buildOutputLog."@en ;
+		sh:path tool:buildOutputLog ;
+	] ;
+	sh:targetSubjectsOf tool:buildOutputLog ;
 	.
 
 tool:buildProject
```

Comment:
@sbarnum - These properties no longer have any references in the `tool:` namespace.  Did you intend for them all to migrate to the `observable:` namespace, and change their types (e.g. some were datatype properties and are now object properties)?
   - `tool:buildProject`
   - `tool:buildScript`
   - `tool:buildUtility`
   - `tool:compilationDate`
   - `tool:compilers`